### DISCUSSION TITLE
Training Log 2023-02-14

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#348] [Training Logs 训练日志]
# CPM-Live Training Log (February, 14)

Time: February, 14 2023 19:00

Recorder: @zh-zheng

## Loss
- Begin: 2.17
- End: 2.17
	
## Completed Data
- $\approx$ 2407.23GB

## Average Grad Norm
- 2.32

## Progress
- 95.36%

## Comment

Whether or not to have a Valentine's Day is a question worth pondering, and CPM-Bee gives a realistic answer.

**Comment by pvop** [comment]
有checkpoint可以下载吗
